src/main/kotlin/no/nav/tiltaksarrangor/config/HttpClientConfig.kt

```diff
@@ -29,6 +29,22 @@ class HttpClientConfig {
 			.build()
 	}
 
+	@Bean
+	fun amtArrangorHttpClient(
+		clientConfigurationProperties: ClientConfigurationProperties,
+		oAuth2AccessTokenService: OAuth2AccessTokenService
+	): OkHttpClient {
+		val registrationName = "amt-arrangor-tokenx"
+		val clientProperties = clientConfigurationProperties.registration[registrationName]
+			?: throw RuntimeException("Fant ikke config for $registrationName")
+		return OkHttpClient.Builder()
+			.connectTimeout(5, TimeUnit.SECONDS)
+			.readTimeout(5, TimeUnit.SECONDS)
+			.followRedirects(false)
+			.addInterceptor(bearerTokenInterceptor(clientProperties, oAuth2AccessTokenService))
```

Comment:
Hjernen min er full av tokens for tiden... Blir dette `oAuth2AccessTokenService.getAccessToken(clientProperties)`  tokenet et client-credentials token, eller bruker det tokenet til den ansatte videre?

Reply:
Her tar jeg inn et tokenx-token og veksler det inn i et nytt tokenx-token som fortsatt har bruker i kontekst, men som er scopet til amt-arrangør :)